src/App.js

```diff
@@ -1,24 +1,22 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { useState } from "react";
+import {  Route, Routes } from "react-router-dom";
+import LoginPage from "./components/Login/Login";
+import Header from './components/Header/header';
+import Home from './components/Home/home'
+import SignUpPage from "./components/Signup/signUp";
 
 function App() {
+ 
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <div>
+      <Header/>
+      <Routes>
+        <Route exact path="/" element={<Home />} />
```

Comment:
why dowe need a HOME component @PuniaMohit

Reply:
we can redirect user to login page if he is not logged in and if he is logged in user will be go to Poll list pages @PuniaMohit